### PR TITLE
dav1d: update to 0.5.0

### PR DESCRIPTION
**Pull request by mustaqimM**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3367,7 +3367,7 @@ libcsfml-window.so.2.5 CSFML-2.5_1
 libcsfml-system.so.2.5 CSFML-2.5_1
 libcsfml-network.so.2.5 CSFML-2.5_1
 libtokyocabinet.so.9 tokyocabinet-1.4.48_1
-libdav1d.so.1 libdav1d1-0.2.0_1
+libdav1d.so.3 libdav1d1-0.5.0_1
```
Comment (daniel-eys) [inline common/shlibs]: Looks like a versioned packagename, so this should be libdav1d3 then to reflect so soversion.

Comment (mustaqimM) [reply common/shlibs]: I don't think so? Since the package name is `libdav1d1` and that would need to be changed.

Comment (daniel-eys) [reply common/shlibs]: See for example the versioned libraries `libssl47`, `libcrypto45`, `libtls19` of the `libressl` package (especially the 2.9.2 update commit 98dd6f95fc09d489a93155bb9d827f3235e160fd)
The new soname mapping needs to be added to `common/shlibs`.

Comment (mustaqimM) [reply common/shlibs]: @daniel-eys It fails to build as you suggested. From what I can see in the [Manual](https://github.com/void-linux/void-packages/blob/master/Manual.md#runtime-dependencies), only the package's minimal version needs to change(along with the new lib version). And packages that require the lib need a revbump: 
```zsh
$> xrevshlib libdav1d1
ffmpeg
```

Comment (daniel-eys) [reply common/shlibs]: I'm not quite sure why the library subpackage of `dav1d` was introduced with a versioned packagename and not just `libdav1d`.
Nevertheless, with versioned packagenames, the appended number should be consistent with the provided soversion. (libdav1d.so.3 <-> libdav1d3)
Yes, ffmpeg needs a revbump to link against the new `libdav1d` soversion.

Comment (mustaqimM) [reply common/shlibs]: okay, can you check it now please.

Comment (Hoshpak) [reply common/shlibs]: You also need to actually create the sub-package. So remove the symlink for libdavid1, create one for libdav1d3, rename the function and dependency in libdav1d-devel.

**Comment by Chocimier** [comment]
ffmpeg should be bumped in separate commit

**Pull request merged**